weldx/transformations/util.py

```diff
@@ -286,3 +292,9 @@ def vector_points_to_left_of_vector(vector, vector_reference):
 
     """
     return int(np.sign(np.linalg.det([vector_reference, vector])))
+
+
+coordinate_types = Union[xr.DataArray, np.ndarray, List]
+types_orientation = Union[xr.DataArray, np.ndarray, List[List], Rot]
+time_types = Union[pd.DatetimeIndex, pd.TimedeltaIndex, pint.Quantity]
+types_time_and_lcs = Union[time_types, "weldx.transformations.LocalCoordinateSystem"]
```

Comment:
Since we started separating stuff into multiple files, what do you think of adding a file for definitions? 

@marscher @CagtayFabry

Reply:
I can add a new module just for the type aliases, if you think that this would be easier to use.